mock/py/mockbuild/backend.py

```diff
@@ -737,18 +737,20 @@ def rebuild_package(self, spec_path, timeout, check, dynamic_buildrequires):
         if additional_opts == ['']:
             additional_opts = []
 
-        def get_command(mode):
+        def get_command(mode, checkdeps=False):
+            nodeps_opt = [] if checkdeps else ['--nodeps']
             command = [self.config['rpmbuild_command']] + mode + \
                       [self._rpmbuild_noclean_option] + \
-                      ['--target', self.rpmbuild_arch, '--nodeps'] + \
+                      ['--target', self.rpmbuild_arch] + nodeps_opt + \
                       check_opt + [spec_path] + additional_opts
             command = ["bash", "--login", "-c"] + [' '.join(command)]
             return command
 
         bd_out = self.make_chroot_path(self.buildroot.builddir)
         max_loops = int(self.config.get('dynamic_buildrequires_max_loops'))
         success = False
-        if dynamic_buildrequires and self.config.get('dynamic_buildrequires'):
+        dynamic_buildrequires = dynamic_buildrequires and self.config.get('dynamic_buildrequires')
```

Comment:
I've repurposed this variable here to say whether dynamic_buildrequires were used. If that's not nice, I can create a new one, or decide later based on the value of `success`.

Reply:
This is fine for sure!